contracts/Witnet.sol

```diff
@@ -491,7 +491,14 @@ library Witnet {
         stageName(error[1]),
         " stage"
       );
-    } else {
+      } else if (errorCode == ErrorCodes.BridgeMalformedRequest) {
+      errorMessage = "The structure of the request is invalid and it can not be parsed";
+      } else if (errorCode == ErrorCodes.BridgePoorIncentives) {
+      errorMessage = "The wit cost of the request exceeds the limit";
+      } else if (errorCode == ErrorCodes.BridgeOversizedResult) {
+      errorMessage = "The request result length exceeds the limit";
+      }
```

Comment:
```suggestion
      } else if (errorCode == ErrorCodes.BridgeMalformedRequest) {
          errorMessage = "The structure of the request is invalid and it cannot be parsed";
      } else if (errorCode == ErrorCodes.BridgePoorIncentives) {
          errorMessage = "The wit cost of the request exceeds the limit";
      } else if (errorCode == ErrorCodes.BridgeOversizedResult) {
          errorMessage = "The request result length exceeds a bridge contract defined limit";
      }
```

Indentation :)

I don't know, the second error message poor incentives is not completely accurate because it could failed because of  the collateral. The documentation in `witnet-rust` for that error was:

```
/// The request is rejected on the grounds that it may cause the submitter to spend or stake an
/// amount of value that is unjustifiably high when compared with the reward they will be getting
```